app/src/main/java/com/nutomic/syncthingandroid/activities/FirstStartActivity.java

```diff
@@ -295,6 +298,16 @@ public Object instantiateItem(ViewGroup container, int position) {
             View view = layoutInflater.inflate(slides[position].layout, container, false);
 
             switch (slides[position]) {
+                case NOTIFICATION:
```

Comment:
You should also implement `shouldSkipSlide()` for `NOTIFICATION`

Reply:
fixed.